### ISSUE TITLE
[Feature Request] Make some parameter untrainable when training on estimator

### ISSUE THREAD

**Issue by aiscientist** [tensorflow/tensorflow#23871]
<em>Please make sure that this is a feature request. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:feature_template</em>

**System information**
- TensorFlow version (you are using): 1.10
- Are you willing to contribute it (Yes/No): Yes

**Describe the feature and the current behavior/state.**
1. In estimator, I am wondering if there are way to make some parameter untrainable when training on estimator.
In session approaches, it's just load tf.trainable_variable() and remove parameters from list before
optimizer process, but I cannot see any features related to those in estimator.

**Will this change the current api? How?**

This is basic features and already implemented on the session method. So, the estimator should support this.

**Comment by karmel** [comment]
I do not fully understand the request here. Can you provide a concise code sample demonstrating the current behavior, and the desired behavior?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 29 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 44 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
We are closing this issue for now due to lack of activity. Please comment if this is still an issue for you. Thanks!
